README.md

```diff
@@ -29,7 +29,7 @@ Generate using CLI
 asyncapi generate fromTemplate <asyncapi.yaml> @asyncapi/markdown-template@1.2.1
 ```
 
-You can replace `<asyncapi.yaml>` with local path or URL pointing to [any AsyncAPI document](https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka.yml).
+You can replace `<asyncapi.yaml>` with local path or URL pointing to any AsyncAPI document like [this one](https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka-asyncapi.yml).
```

Comment:
```suggestion
You can replace `<asyncapi.yaml>` with local path or URL pointing to [any AsyncAPI document](https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka-asyncapi.yml).
```

thanks for fixing link, but the alias needs to be old one from the accessibility perspective and making aliases nicer for screen readers. Screen reader for people with reading disability will just say "this one" which will not be much helpful.